packages/mdx/mdx-hast-to-jsx.js

```diff
@@ -1,12 +1,17 @@
 const {transformSync} = require('@babel/core')
 const uniq = require('lodash.uniq')
-const {serializeTags} = require('remark-mdx/lib/serialize/mdx-element')
-const serializeMdxExpression = require('remark-mdx/lib/serialize/mdx-expression')
+const encode = require('stringify-entities/light')
```

Comment:
The changes in this file are to make micromark and the estrees work, but otherwise not produce noise by keeping stuff as close as it already was. A future PR will compile compile to JS file w/o JSX.